tools/inference/gradio_inference.py

```diff
@@ -98,13 +98,13 @@ def get_inferencer(config_path: Path, weight_path: Path, meta_data_path: Optiona
     inferencer: Inferencer
     if extension in (".ckpt"):
         module = import_module("anomalib.deploy.inferencers.torch")
-        TorchInferencer = getattr(module, "TorchInferencer")
-        inferencer = TorchInferencer(config=config, model_source=weight_path, meta_data_path=meta_data_path)
+        torch_inferencer = getattr(module, "TorchInferencer")
```

Comment:
So with the current design shouldn't this be called `gradio.py`?

Reply:
It would cause some import issues, because at the top of this file we are importing the actual `gradio` package.